From f1ff13a78c8a9c07d962335cfdf900cb5e6e2218 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sat, 16 Jul 2022 19:39:04 +0000
Subject: [PATCH] (build) add missing include after c80cd084921c (#254)

===> Checking for items in STAGEDIR missing from pkg-plist
Error: Orphaned: /man1/aclip.1
===> Checking for items in pkg-plist which are not in STAGEDIR
Error: Missing: share/man/man1/aclip.1.gz
---
 src/tools/acfgfs/CMakeLists.txt     | 2 ++
 src/tools/aclip/CMakeLists.txt      | 2 ++
 src/tools/aloadimage/CMakeLists.txt | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/src/tools/acfgfs/CMakeLists.txt b/src/tools/acfgfs/CMakeLists.txt
index ca1a4e2d..7cdffc9b 100644
--- a/src/tools/acfgfs/CMakeLists.txt
+++ b/src/tools/acfgfs/CMakeLists.txt
@@ -14,6 +14,8 @@ find_package(RT REQUIRED)
 find_package(Threads REQUIRED)
 find_package(DL REQUIRED)
 
+include(GNUInstallDirs) # CMAKE_INSTALL_MANDIR
+
 add_definitions(
 	-Wall
 	-D__UNIX
diff --git a/src/tools/aclip/CMakeLists.txt b/src/tools/aclip/CMakeLists.txt
index 30dee001..615852ea 100644
--- a/src/tools/aclip/CMakeLists.txt
+++ b/src/tools/aclip/CMakeLists.txt
@@ -17,6 +17,8 @@ find_package(RT REQUIRED)
 find_package(Threads REQUIRED)
 find_package(DL REQUIRED)
 
+include(GNUInstallDirs) # CMAKE_INSTALL_MANDIR
+
 add_definitions(
 	-Wall
 	-D__UNIX
diff --git a/src/tools/aloadimage/CMakeLists.txt b/src/tools/aloadimage/CMakeLists.txt
index b9bce35e..ced8f2c9 100644
--- a/src/tools/aloadimage/CMakeLists.txt
+++ b/src/tools/aloadimage/CMakeLists.txt
@@ -19,6 +19,8 @@ find_package(DL REQUIRED)
 find_package(Threads REQUIRED)
 find_package(Sanitizers REQUIRED)
 
+include(GNUInstallDirs) # CMAKE_INSTALL_MANDIR
+
 SET(LIBRARIES
 	Math::Math
 	RT::RT
-- 
2.37.1

